src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -2475,8 +2473,8 @@ public enum MysteriousTreasures implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               flip {
-                opponentCantPlaySupporterNextTurn(delegate) //TODO: Que el static use ${thisMove.name}
-                bc "${thisMove.name} - Supporters can't be played from the turn player's hand during the next turn" //TODO: Find way to use "Player A/B" as a $variable
+                opponentCantPlaySupporterNextTurn(delegate)
+                bc "${thisMove.name} - Supporters can't be played from the turn player's hand during the next turn" //TODO: Move this to the static.
```

Comment:
Shouldn't this already be done with the changes to the static in this PR?

Reply:
Yeah, could do it already. Will commit it in a bit 👍